packages/debug/src/browser/debug-command.ts

```diff
@@ -121,5 +165,81 @@ export class DebugCommandHandlers implements MenuContribution, CommandContributi
             isEnabled: () => true,
             isVisible: () => true
         });
+
+        registry.registerCommand(DEBUG_COMMANDS.SUSPEND_ALL_THREADS);
+        registry.registerHandler(DEBUG_COMMANDS.SUSPEND_ALL_THREADS.id, {
+            execute: () => {
+                const debugSession = this.debugSessionManager.getActiveDebugSession();
+                if (debugSession) {
+                    debugSession.threads().then(response => {
```

Comment:
I would move these code to DebugSession in function `suspend(threadId?: number)`
The same for resume command.;